tests/hooks/event-conversion-utils.spec.ts

```diff
@@ -707,9 +803,426 @@ describe("formatDates", () => {
+      },
+    });
+
+    eventsAirtableRecords.getEventsQuery = getEventsQueryMock;
+    eventsAirtableRecords.isEventInDateRange = vi.fn().mockReturnValue(true);
+
+    (axios.get as any).mockResolvedValue({
+      data: "Omnis non molestiae voluptatem beatae soluta nobis ducimus.",
+    });
+    const result = await eventsAirtableRecords.fetchSeminarSeriesEvents(-10);
+
+    expect(result).toStrictEqual([
+      {
+        endDate: "",
+        institution: "Kihn - Rolfson",
+        location: "YouTube",
+        speaker: "Deondre Halvorson",
+        title: "Ea commodi earum id.",
+        to: "http://jam-packed-unity.com",
+        abstract: "Qui facere assumenda ducimus.",
+        date: "June 9, 2023",
+        image: "/images/events/downloaded/undefined.jpg",
+        startDate: "2023-06-09",
+      },
+      {
+        endDate: "",
+        institution: "Okuneva Group",
+        location: "YouTube",
+        speaker: "Lela Bahringer",
+        title: "delectus eaque consectetur",
+        to: "http://overlooked-sleeping.org",
+        abstract: "",
+        date: "July 29, 2022",
+        image: "/images/events/downloaded/undefined.jpg",
+        startDate: "2022-07-29",
+      },
+    ]);
+  });
+
+  test("If no recordFields should get from getAllFieldNames", async () => {
+    const eventsAirtableRecords = new FakeEventsAirtableRecords(
+      "testApiKey",
+      "testView"
+    );
+
+    const getEventsQueryMock = vi.fn().mockReturnValue({
+      eachPage: (callBack: (records: any, nextPage: any) => void) => {
+        callBack(fakeRecords, () => {});
+
+        return Promise.resolve();
+      },
+    });
+
+    const getAllFieldNamesMock = vi.fn().mockReturnValue(mockRecordFields);
+
+    eventsAirtableRecords.getEventsQuery = getEventsQueryMock;
+    eventsAirtableRecords.getAllFieldNames = getAllFieldNamesMock;
+
+    await eventsAirtableRecords.fetchSeminarSeriesEvents(10);
+
+    expect(eventsAirtableRecords.recordFields).toStrictEqual(mockRecordFields);
+  });
+
+  test("test fallback cases", async () => {
+    const eventsAirtableRecords = new FakeEventsAirtableRecords(
+      "testApiKey",
+      "testView",
+      undefined,
+      {}
+    );
+
+    const seminarEvent =
+      await eventsAirtableRecords.convertToSeminarSeriesEvent(fakeRecords[0]);
+
+    expect(seminarEvent).toStrictEqual({
+      abstract: "",
+      date: "TBD",
+      endDate: "",
+      image: "/images/events/no-picture.jpg",
+      institution: "",
+      location: "TBD",
+      speaker: "",
+      startDate: "",
+      title: "",
+      to: "",
+    });
+  });
+});
+
+describe("formatTime", () => {
+  test("format a valid date", () => {
+    const eventsAirtableRecords = new FakeEventsAirtableRecords(
+      "testApiKey",
+      "testView"
+    );
+
+    eventsAirtableRecords.recordFields = {};
+
+    const date = eventsAirtableRecords.formatTime("2020-10-10 00:00:00 UTC");
+
+    expect(date.replace("\u202F", " ")).toBe("12:00 AM UTC");
```

Comment:
Is there maybe a way of writing this assertion without modifying the value returned from the tested method (`date`)? Otherwise it feels _hacky_.

Reply:
Is an issue about platforms (related to encoding). On my local, date has this character `\u202F` but in CI it doesn't

Reply:
It makes sense, thank you for the explanation.

Maybe we could go for one of these approaches:

1. Add a comment to describe why we need to do this `replace` to avoid confusion in the future.
2. Change the assertion to [`toContain`](https://jestjs.io/docs/expect#tocontainitem) instead of `toBe`. (If this works, this would be my personal preference)

Reply:
Great! thanks for those tips